packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -576,25 +576,27 @@ void main() {
     await tester.pumpWidget(
       TestApp(
         textDirection: TextDirection.ltr,
-        child: Material(
-          child: DropdownButtonFormField<String>(
-            key: buttonKey,
-            value: value,
-            onChanged: onChanged,
-            items: menuItems.map<DropdownMenuItem<String>>((String item) {
-              return DropdownMenuItem<String>(
-                key: ValueKey<String>(item),
-                value: item,
-                child: Text(item, key: ValueKey<String>('${item}Text')),
-              );
-            }).toList(),
+        child: Align(
```

Comment:
What's the reason for adding the `Align` (and why not use `Center` instead)?

If the test won't pass without it, it makes me wonder if this isn't a more breaking change that we should consider more carefully.

Reply:
I suppose this is similar to why `ElevatedButton` (which uses `InkWell`) needs a `Center`/`SizedBox` widget as its parent in almost all of its tests. Although I am unsure if this is a breaking change or not.

Reply:
That's a good example, yes. You will need constraints anywhere `Material` is implemented otherwise, widgets that depend on Material expanded to Material unconstrained size such InkWell here

Reply:
OK, thanks, that makes sense.